### DISCUSSION TITLE
[BACKEND] Links in the menu vs Page titles

### DISCUSSION THREAD

**Discussion by addison74** [OpenMage/magento-lts#4216] [Ideas]
I noticed in the Backend that there are differences between the name of the links in the main menu and the title of the page.

| Menu Link  | Page Title |
| ------------- | ------------- |
| Terms and Conditions  | Manage Terms and Conditions |
| Manage Categories | Categories |
| Search Terms | Search |
| Pages | Manage Pages |
| Widgets | Manage Widget Instances |

In the Reports menu there are several differences between what is intended to be the name of the link and that of the page.

Maybe a review of the interface in this regard should be taken into account.

**Comment by addison74** [comment]
Related to this issue, the word "Manage" must also be revised here in menus and page titles. A PR was proposed in the past but the change was not made in the local files and it was closed.

**Comment by sreichel** [comment]
PR welcome.
